test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionProgressIT.java

```diff
@@ -148,6 +230,10 @@ private void checkRunning() throws TException {
               progressList.add(EC_PROGRESS.HALF);
             } else if (rci.progress > 75 && rci.progress <= 100) {
               progressList.add(EC_PROGRESS.THREE_QUARTERS);
+            } else {
+              progressList.add(EC_PROGRESS.INVALID);
+              log.warn("An invalid progress {} has been seen. This should never occur.",
+                  rci.progress);
```

Comment:
I think it would be nice to either fail the test here instead of adding it to the list, or where all the progress states are checked at the end, check them all by using Junits `assertAll()` that way it will alert us if multiple asserts fail instead of just stop at the first failure.

Reply:
Changed to use `assertAll()`. Could have it fail for that specific check there, but we already need to add to a list for the other checks (or at least some way to signal that those progresses have been seen), so I think it is more consistent to just add INVALID to the list. This way also won't end the progress early, which may be better for debugging to see how high the progress goes.

Reply:
Yea I agree. I like checking them all instead of failing early in this case.